weaviate/collection/classes/config.py

```diff
@@ -59,12 +131,38 @@ class Vectorizer(str, Enum):
 
 
 class GenerativeSearches(str, Enum):
+    """The available generative search modules in Weaviate.
+
+    Attributes:
+        `OPENAI`
+            OpenAI's Generative module.
+        `COHERE`
+            Cohere's Generative module.
+        `PALM`
+            PaLM's Generative module.
+    """
+
     OPENAI = "generative-openai"
     COHERE = "generative-cohere"
     PALM = "generative-palm"
 
 
 class VectorDistance(str, Enum):
+    """Vector similarity distance metric to be used in the `VectorIndexConfig` class.
```

Comment:
Propose to add `Must be the same the distance metric as that used by the vectorizer model.`

Reply:
What happens if they don't use the same metric? Are their searches negatively effected or does Weaviate error?

Reply:
You get suboptimal results, because the neural network is trained to produce optimal results for a metric but now the results are being measured against another.

My analogy is running a 100m race and then determining a winner by height 😅